EventHorizon/Core/Options.lua

```diff
@@ -671,10 +697,16 @@ function EventHorizon:CreateDebuffSpellsOptions()
 		debuffSpells[k] = {
 			order = count,
 			name = k,
-			icon = select(3,GetSpellInfo(k)),
+			icon = self.spellCache.GetBestIconMatchingName(k),
 			type = "group",
 			width = "half",
 			args = {
+				spellId = {
+					order = 1,
+					name = "SpellID: "..EventHorizon.opt.debuffs[k].spellId,
+					type = "description",
+					width = "full"
+				},
 				enabled = {
 					order = 1,
```

Comment:
Looks like the order needs to be incremented for all these args with the inclusion of spellId as order=1

Reply:
good catch